docs/photutils/aperture.rst

```diff
@@ -461,22 +461,21 @@ simply requires the definition of a new `~photutils.Aperture`
 subclass.
 
 All `~photutils.Aperture` subclasses must implement only two methods,
-``do_photometry(data)`` and ``plot()``.  They can optionally
-implement a third method, ``area()``.
+``do_photometry(data)`` and ``plot()``, and an ``area``
+`lazyproperty`_.
 
 * ``do_photometry(data)``: A method to sum the pixel values within the
   defined aperture.
 * ``plot()``: A method to plot the aperture on a `matplotlib`_ Axes
   instance.
-* ``area()``: If convenient to calculate, this returns the area of the
-  aperture.  This speeds computation in certain situations (such as a
-  scalar error).
+* ``area()``: A `lazyproperty`_ to return the area of the aperture.
 
 Note that all x and y coordinates here refer to the fast and slow
 (second and first) axis of the data array respectively.  See
 :ref:`coordinate-conventions`.
 
 .. _matplotlib: http://matplotlib.org
+.. _lazyproperty: http://astropy.readthedocs.org/en/stable/api/astropy.utils.misc.lazyproperty.html?highlight=lazyproperty
```

Comment:
Can you make this an intersphinx link like we do for all other functions / classes from Astropy?
```
`~astropy.utils.lazyproperty`
```
I see this has moved form `misc.py` to `decorators.py` ... it's now here:
http://astropy.readthedocs.org/en/latest/api/astropy.utils.decorators.lazyproperty.html?highlight=lazyproperty

But this intersphinx-link should work for the old or new version, no?